tests/lib/rules/avoid-leaking-state-in-ember-objects.js

```diff
@@ -11,7 +11,7 @@ const RuleTester = require('eslint').RuleTester;
 
 describe('imports', () => {
   it('should expose the default ignored properties', () => {
-    expect(rule.DEFAULT_IGNORED_PROPERTIES).toEqual([
+    expect(rule.DEFAULT_IGNORED_PROPERTIES).toStrictEqual([
```

Comment:
what's the reason to use `Strict` here?

Reply:
Prefer the more strict assertion when possible.

https://github.com/jest-community/eslint-plugin-jest/blob/master/docs/rules/prefer-strict-equal.md

Reply:
hmm, that seems irrelevant to me in this case though 🤔

Reply:
Yeah in this case it doesn't matter but it's still valuable to encourage stricter assertions in general.